server/plugin/webhook.go

```diff
@@ -826,8 +826,11 @@ func (p *Plugin) postPullRequestReviewEvent(event *github.PullRequestReviewEvent
 
 	switch event.GetReview().GetState() {
```

Comment:
2/5 it's simpler to convert to upper case

```suggestion
	switch strings.ToUpper(event.GetReview().GetState()) {
```